powertools-parameters/src/test/java/software/amazon/lambda/powertools/parameters/internal/LambdaParametersAspectTest.java

```diff
@@ -0,0 +1,57 @@
+package software.amazon.lambda.powertools.parameters.internal;
+
+import org.junit.jupiter.api.Test;
+import software.amazon.lambda.powertools.parameters.Param;
+import software.amazon.lambda.powertools.parameters.exception.TransformationException;
+import software.amazon.lambda.powertools.parameters.transform.Base64Transformer;
+import software.amazon.lambda.powertools.parameters.transform.JsonTransformer;
+import software.amazon.lambda.powertools.parameters.transform.ObjectToDeserialize;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class LambdaParametersAspectTest {
+
+    @Param(key = "/simple", provider = CustomProvider.class)
+    private String param;
```

Comment:
Possible to have a test validating default config with just the key ?

Reply:
it is possible but it's tricky to mock. I'll add it.